modules/ob-about-builds.adoc

```diff
@@ -4,24 +4,24 @@
 
 :_content-type: CONCEPT
 [id="about-builds_{context}"]
-= About the Build resource
+= Build resource
 
 The `Build` resource defines the source code of your application and the location where your application images will be pushed. The following example shows a simple build that consists of a Git source, a build strategy, and an output image:
 
 [source,yaml]
 ----
-apiVersion: shipwright.io/v1beta1
-kind: Build
-metadata:
-  name: kaniko-golang-build
-spec:
-  source:
-    git:
-      url: https://github.com/username/taxi
-  strategy:
-    name: kaniko
-    kind: ClusterBuildStrategy
-  output:
+apiVersion: shipwright.io/v1beta1 
```

Comment:
Please trim the whitespace added on each of these lines as per our [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-file-guidelines).